libpod/oci_conmon_common.go

```diff
@@ -1316,6 +1316,10 @@ func (r *ConmonOCIRuntime) configureConmonEnv() ([]string, error) {
 			// The NOTIFY_SOCKET must not leak into the environment.
 			continue
 		}
+		if strings.HasPrefix(v, "DBUS_SESSION_BUS_ADDRESS=") && !rootless.IsRootless() {
+			// The DBUS_SESSION_BUS_ADDRESS must not leak into the environment.
```

Comment:
This comment seems wrong given we still need it for rootless.
Also some actual reason why would be great to have in the comment.

Reply:
fixed now